.buildkite/publish/publish-common.sh

```diff
@@ -24,6 +24,3 @@ fi
 export BASE_TAG_NAME=${DOCKER_IMAGE_NAME:-docker.elastic.co/enterprise-search/elastic-connectors}
 export DOCKERFILE_PATH=${DOCKERFILE_PATH:-Dockerfile}
 export DOCKER_ARTIFACT_KEY=${DOCKER_ARTIFACT_KEY:-elastic-connectors-docker}
-
-export VAULT_ADDR=${VAULT_ADDR:-https://vault-ci-prod.elastic.dev}
-export VAULT_USER="docker-swiftypeadmin"
```

Comment:
Nice benefit is that we don't need to look up docker creds anymore, so no generic usage of vault secrets